breadcord/module.py

```diff
@@ -51,6 +51,9 @@ def __repr__(self) -> str:
 
     async def load(self) -> None:
         self.load_settings_schema()
+        # Some modules might fail to load because of their settings. We want to save them so the user can fix them
+        self.bot.save_settings()
```

Comment:
I'm tentative of "implicit saves" of settings, because of the potential of overwriting uncommitted user changes to the settings file if done too aggressively - an effort to save the user's changes might have the opposite effect. It's been so long since I've touched Breadcord settings that I'd have to discuss this further with you probably.

Reply:
The thing is, we don't really have a choice. Either we save the settings or the user has no settings to edit